x-pack/legacy/plugins/reporting/server/core.ts

```diff
@@ -95,13 +99,13 @@ export class ReportingCore {
     return (await this.getPluginStartDeps()).enqueueJob;
   }
 
-  public async getBrowserDriverFactory(): Promise<HeadlessChromiumDriverFactory> {
-    return (await this.getPluginSetupDeps()).browserDriverFactory;
```

Comment:
the `browserDriverFactory` object is no longer passed around between modules, so this method isn't needed anymore.